### ISSUE TITLE
Ghost blocks create X-Ray effect when disguised as Chisel blocks [1.12]

### ISSUE THREAD

**Issue by Kevin67632342** [AbrarSyed/SecretRoomsMod-forge#251]
No Optifine Installed.
Using Version 1.12.2-5.6.4

When disguising a ghost block with a chisel block it allows players to see through the world.

Works with all chisel blocks. Using Chisel 1.12.2-0.2.0.31

![2018-06-01_23 15 57](https://user-images.githubusercontent.com/36777444/40870314-df4308bc-65fa-11e8-9c13-72010fcd771b.png)
![2018-06-01_23 16 05](https://user-images.githubusercontent.com/36777444/40870315-df551994-65fa-11e8-985a-e3aa1729cb5f.png)

**Comment by Wyn-Price** [comment]
Would this only happen with ghost blocks, as you can walk in them?

**Comment by Silver-Wulf** [comment]
Yes, sorry. I'm with him and it is only when you are inside one of the ghost blocks.

**Comment by Wyn-Price** [comment]
hmm ok, i can fix this soon. At the moment i got exams so its hard for me to keep up

**Comment by Brycey92** [comment]
@Wyn-Price this actually happens with all non-solid chisel blocks, as I discussed in the issue tterag1098 linked directly above. ^
